sources/packages/forms/src/form-definitions/studentrequestchange.json

```diff
@@ -0,0 +1,696 @@
+{
```

Comment:
Please ensure that the below option is checked.
![image](https://user-images.githubusercontent.com/61259237/159767874-9b7266c8-3396-45be-971b-102ad121a3de.png)

@ann-aot @henryoforeh-aot @guru-aot just a reminder, always that we have a "list of values" like dropdowns, checkboxes, radio button lists, please ensure that the above option is checked.